pkgs/top-level/all-packages.nix

```diff
@@ -12043,7 +12043,7 @@ with pkgs;
     ];
     extraConfig = import ../os-specific/linux/kernel/hardened-config.nix {
       inherit stdenv;
-      inherit (linux) version;
+      inherit (linux_hardened_copperhead) version;
```

Comment:
I'm fairly certain copperhead uses stricter defaults than vanilla, so if we standardise on it some of what has been set above may be strictly redundant.